src/index.js

```diff
@@ -5,8 +5,18 @@ const TYPE_IMPORT = 'Import';
 
 const buildImport = template(`
   (new Promise((resolve) => {
-    require.ensure([], (require) => {
-      resolve(require(SOURCE));
+    require.ensure([SOURCE], (require) => {
+      var ns = require(SOURCE);
+      
+      if (ns && ns.__esModule) {
+        resolve(ns);
+      } else {
+        var wrap = Object.create(null);
```

Comment:
Right - pre-ES5 and pre-`__proto__`, it's impossible to have a null prototype object, so it'd need to just be a normal object there.